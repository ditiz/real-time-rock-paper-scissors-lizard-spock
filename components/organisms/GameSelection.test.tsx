import "@testing-library/jest-dom"
import { render, screen } from "@testing-library/react"
import GameSelectionOrganism from "./GameSelection"

it("renders GameSelectionOrganism unchanged", () => {
  const { container } = render(<GameSelectionOrganism />)
  expect(container).toMatchSnapshot()
})
