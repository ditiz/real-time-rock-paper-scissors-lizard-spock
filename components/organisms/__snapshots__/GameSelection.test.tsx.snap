// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders GameSelectionOrganism unchanged 1`] = `
<div>
  <section
    class="game-selection"
  >
    <article
      class="card"
    >
      <a
        href="/Classic"
      >
        <div>
          <h2>
            Game Classic
          </h2>
          <p>
            Classic game of rock, paper, scissors
          </p>
        </div>
      </a>
    </article>
    <article
      class="card"
    >
      <a
        href="/Advanced"
      >
        <div>
          <h2>
            Game Advanced
          </h2>
          <p>
            Game of rock, paper, scissors, lizard, spock
          </p>
        </div>
      </a>
    </article>
  </section>
</div>
`;
